src/repl/completion.rs

```diff
@@ -1,4 +1,4 @@
-//! User text completion for REPL interations.
+//! User text completion for REPL inputs.
```

Comment:
I guess this was meant to say "iterations" (and not "interactions"). I just changed it to "inputs" since I think that makes even more sense than iterations.